### DISCUSSION TITLE
Shares scheduled scans

### DISCUSSION THREAD

**Discussion by janaxhell** [nicotine-plus/nicotine-plus#3180] [Q&A] [answered]
I have Nicotine+ deployed as docker container. Every day it scans my shares, taking at least an hour of thrashing my disk. Is there a way to set the scan to a different time, or totally manual? I can't find any option in the GUI, nor references here in the Readme.

**Comment by slook** [comment]
Such automatic rescanning functionality isn't implemented yet, see #2900, and therefore your container seems to be restarting itself for some reason such as the program crashing maybe due to some external event or fault occurring daily.

You could confirm this by checking the timestamp of the earliest log entry displayed on the first line in the Log Pane, or enabling logging to file in config if you are running headless with no GUI.

**Reply by janaxhell** [reply]
Ahh, then it's probably not crashing, but restarting after Watchtower updates, which is basically daily + (noticed right now) the "Rescan shares on startup" option enabled in the Preferences. I assume disabling it will prevent Nicotine from rescanning each time a new build is deployed. Is that so?
EDIT Strange, I see last Release is 3 weeks ago, but Watchtower using ghcr.io/fletchto99/nicotine-plus-docker:latest updates it every day. Checking WT logs, this is happening with 3 specific apps:
```
time="2024-10-27T09:21:25Z" level=info msg="Creating /jackett"
time="2024-10-27T09:21:29Z" level=info msg="Creating /nicotine-plus"
time="2024-10-27T09:21:36Z" level=info msg="Creating /monitoring-grafana"
```
All other apps are updated normally, now and then, not every day. Any idea of what could cause this? Nicotine+ is very recent, I've added it to my server less than 2 months ago, while jackett and grafana have been there for more than 2 years. Nevertheless they have this same issue.
EDIT 2
Does Nicotine+ have nightlies overlapping the :latest tag? I was suggested on Reddit that this could happen for apps that have no official release, but actually update builds every night.

**Reply by slook** [reply, marked as answer]
> Is that so?

Yes, that should workaround your issue for now.

> Any idea of what could cause this?

I don't have any clue, sorry. Docker support is experimental by third-parties.

> Does Nicotine+ have nightlies overlapping the :latest tag?

No, your "[nicotine-plus-docker](https://github.com/fletchto99/nicotine-plus-docker)" docker image is based on an old stable release and has nothing to do with the main nicotine-plus repo at this time, so you would have to ask its maintainer _fletchto99_ about helping with debugging and updating that, as such there are not yet any requests for GitHub Actions integrations for automated package deployment. However, it looks like issues (5 closed to date) are being attended to in that repo at https://github.com/fletchto99/nicotine-plus-docker/issues

Alternately or in addition you might test the "[nicotineplus-proper](https://github.com/sirjmann92/nicotineplus-proper)" docker image by _sirjmann92_ which was updated 3 days ago and has 10 closed issues to date. It has a different backend I think, Broadway, and there are perhaps other important reasons for having separate choices of image, although I've no clue what exactly.

And the "[nicotine-docker](https://github.com/cproensa/nicotine-docker)" docker image by _cproensa_ also exists using noVNC and was updated yesterday, but that also uses the stable release. It doesn't look like theres any which uses latest unstable from master (if there was then it would be likely to break all the time, because the nicotine-plus application isn't a stable API, unfortunately).

Eventually [a desire exists](https://github.com/nicotine-plus/nicotine-plus/discussions/2312) for building development and ongoing package maintenance will be synchronized and merged into the team, once those devs can find enough testers in the community to motivate their much appreciated efforts.

We're happy to help out in whatever way we can if and when required, but for now any further packaging related questions you'll need to make Issues on their repo(s) for them to help you directly (and in turn, you will then of course be helping everyone make progress by testing the package(s) in your particular environment(s)).

**Reply by janaxhell** [reply]
Thank you so much for the extensive explanation, I had no idea there were so many docker variants, I guess I've adopted the first that appeared while looking for it. I'll check those and see what happens with Watchtower. Cheers!
